### ISSUE TITLE
EAP 2017.2 fatal error

### ISSUE THREAD

**Issue by smeevil** [dkandalov/live-plugin#69]
Hi,

The plugin seems to crash during startup with the latest EAP.

The error message is as follows :

```From listener liveplugin.toolwindow.PluginToolWindowManager$1@5948270f (class liveplugin.toolwindow.PluginToolWindowManager$1)```

Stacktrace :

```From listener liveplugin.toolwindow.PluginToolWindowManager$1@5948270f (class liveplugin.toolwindow.PluginToolWindowManager$1)
java.lang.NullPointerException
	at com.intellij.openapi.wm.impl.ToolWindowManagerImpl.a(ToolWindowManagerImpl.java:1754)
	at com.intellij.openapi.wm.impl.ToolWindowManagerImpl.a(ToolWindowManagerImpl.java:1169)
	at com.intellij.openapi.wm.impl.ToolWindowManagerImpl.registerToolWindow(ToolWindowManagerImpl.java:1124)
	at com.intellij.openapi.wm.impl.ToolWindowManagerImpl.registerToolWindow(ToolWindowManagerImpl.java:1113)
	at liveplugin.toolwindow.PluginToolWindowManager$PluginToolWindow.registerWindowFor(PluginToolWindowManager.java:229)
	at liveplugin.toolwindow.PluginToolWindowManager$1.projectOpened(PluginToolWindowManager.java:140)
	at com.intellij.openapi.project.impl.ProjectManagerImpl$1.projectOpened(ProjectManagerImpl.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.util.messages.impl.MessageBusConnectionImpl.deliverMessage(MessageBusConnectionImpl.java:117)
	at com.intellij.util.messages.impl.MessageBusImpl.doPumpMessages(MessageBusImpl.java:443)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpWaitingBuses(MessageBusImpl.java:404)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:393)
	at com.intellij.util.messages.impl.MessageBusImpl.sendMessage(MessageBusImpl.java:374)
	at com.intellij.util.messages.impl.MessageBusImpl.access$200(MessageBusImpl.java:45)
	at com.intellij.util.messages.impl.MessageBusImpl$2.invoke(MessageBusImpl.java:231)
	at com.sun.proxy.$Proxy25.projectOpened(Unknown Source)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.c(ProjectManagerImpl.java:747)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.l(ProjectManagerImpl.java:389)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:163)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:86)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:109)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:416)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:399)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.h(IdeEventQueue.java:821)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:649)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:365)
	at com.intellij.ide.IdeEventQueue.pumpEventsForHierarchy(IdeEventQueue.java:923)
	at com.intellij.openapi.progress.util.ProgressWindow.startBlocking(ProgressWindow.java:215)
	at com.intellij.openapi.progress.util.ProgressWindow.startBlocking(ProgressWindow.java:202)
	at com.intellij.openapi.application.impl.ApplicationImpl.runProcessWithProgressSynchronously(ApplicationImpl.java:616)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcessWithProgressSynchronously(CoreProgressManager.java:414)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.runProcessWithProgressSynchronously(ProgressManagerImpl.java:117)
	at com.intellij.openapi.progress.impl.CoreProgressManager.b(CoreProgressManager.java:332)
	at com.intellij.openapi.progress.impl.CoreProgressManager.run(CoreProgressManager.java:317)
	at com.intellij.openapi.progress.ProgressManager.run(ProgressManager.java:190)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.loadAndOpenProject(ProjectManagerImpl.java:475)
	at com.intellij.ide.impl.ProjectUtil.openProject(ProjectUtil.java:190)
	at com.intellij.ide.impl.ProjectUtil.openOrImport(ProjectUtil.java:116)
	at com.intellij.ide.CommandLineProcessor.a(CommandLineProcessor.java:58)
	at com.intellij.ide.CommandLineProcessor.processExternalCommandLine(CommandLineProcessor.java:197)
	at com.intellij.idea.IdeaApplication$IdeStarter.a(IdeaApplication.java:320)
	at com.intellij.idea.IdeaApplication$IdeStarter.a(IdeaApplication.java:362)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:86)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:109)
	at com.intellij.openapi.application.TransactionGuardImpl.submitTransaction(TransactionGuardImpl.java:118)
	at com.intellij.openapi.application.TransactionGuard.submitTransaction(TransactionGuard.java:122)
	at com.intellij.idea.IdeaApplication$IdeStarter.main(IdeaApplication.java:361)
	at com.intellij.idea.IdeaApplication.b(IdeaApplication.java:206)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:193)
	at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:206)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.idea.MainImpl$1$1.a(MainImpl.java:70)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:345)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)```

**Comment by dkandalov** [comment]
Hi,

The issue should be fixed in the latest plugin version 0.5.13.

Thanks a lot for reporting this!
I don't always use the latest EAP and might not notice these problems for a while.

**Comment by smeevil** [comment]
Awesome, thanks for taking the time :)
